### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by aavinseth** [microsoft/terminal#6817]
# Environment

```none
Windows build number: Version 2004 (OS Build 19041.329)
Windows Terminal version (if applicable): 1.0.1811.0

Any other software?
```

# Steps to reproduce

1. Download Windows Terminal from Microsoft Store
2. Open Windows Terminal from Start Menu

# Expected behavior

Windows Terminal should open in maximize mode

# Actual behavior

Windows Terminal crash on open

https://aka.ms/AA8wuqk

**Comment by WSLUser** [comment]
I was unable to get the Preview to update from the Store but I manually updated it and suffered no crashes. Also it doesn't open in maximum mode but more like "medium" that can be resized or set the maximum mode.

**Comment by DHowett** [comment]
There's no reason terminal would default to maximized mode. I think you've left out one or two of the steps in the description. Can you help clarify?

Are you in tablet mode? What sort of hardware are you using? Are you in power saver mode? Have you changed any terminal settings?

**Comment by DHowett** [comment]
Unfortunately, your feedback item didn't include any crashes... which means that Terminal might be shutting down on its own (and not crashing!) ☹️

**Comment by aavinseth** [comment]
@DHowett there was a update in Microsoft Store, so as usual I update it. After updating when I was trying to open the app it was shutting down or get crash. Regarding to maximize mode I mis spell it. After this issue, I have uninstalled and install it multiple times but still same thing. Even I have tried preview app but still same problem. Then I installed it again and just restart my system. After restarting, its working fine now.
